README.md

```diff
@@ -33,7 +33,7 @@ A parser for .ls files (providing an index for .nar files)
 ## `pkg/nar/narinfo`
 A parser and generator for `.narinfo` files.
 
-## `pkg/nixbase32`
+## `nixbase32`
```

Comment:
Why is this moved out of `pkg/`?

The idea for moving it into `pkg/` was in https://github.com/nix-community/go-nix/issues/10.

Also, please make the part moving it and changing things in there two separate commits, so the changes in the package itself can be easier reviewed.

Reply:
See this [blog post on layout](https://eli.thegreenplace.net/2019/simple-go-project-layout-with-modules/) for rationale. golang-standards/project-layout is somewhat notorious for self-proclaiming that their layout recommendation is standard without actually having buy-in from the community, sadly.

That said, makes sense to have that be a separate PR. Reverted the change for now.

Reply:
Interesting read, thanks! I'm not opposed to move things around. Can you squash your changes though, so this gets easier to review on a commit-by-commit basis?